startupscript/post-startup.sh

```diff
@@ -14,13 +14,15 @@ readonly USER_NAME="${1}"
 readonly WORK_DIRECTORY="${2}"
```

Comment:
Exporting the variables that are used by the scripts that are called later. This is because those scripts cannot be sourced if we support retries.
Exporting the variables will make the variables available to those scripts.

Reply:
i see... can we do it per line.

like readonly USER_NAME
export USER_NAME

so when someone wants to delete or change the variable name, it's harder to miss.